experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -45,6 +45,10 @@ export const fuzzReducer = combineReducersAsync<Operation, DDSFuzzTestState<Shar
 		applyUndoRedoEdit(tree, contents);
 		return state;
 	},
+	synchronize: async (state) => {
+		applySynchronizationOp(state);
```

Comment:
don't think this line ever gets called since you re-used the exact same op type as `Synchronize` from within the fuzz harness.

I'd either use a different type, or import `Synchronize` from `@fluid-internal/test-dds-utils` and change the processing logic here (which you'll need to have to pass typechecking) to just `fail` with an informative error.

Reply:
changed type from `synchronize` to `synchronizeTrees`

Reply:
might be worth documenting why this is duplicated with harness code. otherwise looks good